rama-http/src/matcher/subdomain_trie.rs

```diff
@@ -0,0 +1,173 @@
+use trie_rs::TrieBuilder;
+use crate::Request;
+use rama_core::{Context, context::Extensions, matcher::Matcher};
+use rama_net::http::RequestContext;
+use rama_net::address::{Host};
+
+#[derive(Debug, Clone)]
+pub struct SubdomainTrieMatcher {
+    trie: trie_rs::Trie<u8>,
```

Comment:
```suggestion
    trie: Trie<u8>,
```

can just also add it to your `use trie_rs::` statement above